src/index.html

```diff
@@ -3,10 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <title>S.W.E.A.T</title>
+    <link rel="stylesheet" href="main.css">
 </head>
 <body>
     <p>Hello World</p>
```

Comment:
I will keep the Hello World Paragraph just so there is something on the page before we decide on a structure :) :+1: